ec2/spark_ec2.py

```diff
@@ -340,6 +346,27 @@ def is_active(instance):
     "t2.small":    "hvm",
 }
 
+# Return correct versions of Spark and Tachyon, given the supplied Spark version
+def get_spark_tachyon_version(opts):
+    spark_tachyon_map = {
+        "0.7.3": "",
+        "0.8.0": "",
+        "0.8.1": "",
+        "0.9.0": "",
+        "0.9.1": "",
+        "1.0.0": "0.4.1",
+        "1.0.1": "0.4.1",
+        "1.0.2": "0.4.1",
+        "1.1.0": "0.5.0",
+        "1.1.1": "0.5.0",
+        "1.2.0": "0.5.0",
+        "1.2.1": "0.5.0",
+    }
+    version = opts.spark_version.replace("v", "")
```

Comment:
Nit: I know git hashes can't contain the letter "v", but it seem wrong to remove all occurrences of the letter. Instead, we probably want to use `lstrip` here.